rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
```

Comment:
yeah good point, I guess that PARAMETER_NOT_SET would be a good default. Not sure how we refer to message constants from another msg file though.